src/server/api/general.js

```diff
@@ -573,6 +573,67 @@ export default function useGeneralApi(app) {
             this.status = 500;
         }
     });
+
+    router.post('/isTosAccepted', koaBody, function*() {
+        const params = this.request.body;
+        const { csrf, username } =
+            typeof params === 'string' ? JSON.parse(params) : params;
+        if (!checkCSRF(this, csrf)) return;
+
+        if (!username) {
```

Comment:
yeah it's inconsistent. the serverApiLogin call at https://github.com/steemit/condenser/blob/master/src/app/redux/UserSaga.js#L370 runs async.

Reply:
My concern is that this implementation would allow someone to iterate over all blockchain accounts and check if they accepted ToS.

Reply:
An alternative could be to implement a `is_tos_accepted` RPC call in conveyor the client can call directly with their posting auth

Reply:
this would make the most sense. i'm working with the available endpoints in conveyor though. the api calls to condenser are rate limited.

Reply:
if we're concerned about a logged-in user trying to suck this data out, we can add a signed call to conveyor. otherwise i think my solution is good for now.